### PR TITLE
Reduce eslint warnings [27]

### PR DESCRIPTION
**Pull request by RedSparr0w**
reduce the number of warnings in eslint `37 → 27`

### CODE REVIEW

File: src/scripts/Notifier.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint @typescript-eslint/camelcase: off */
```
Comment (Ishadijcks) [inline src/scripts/Notifier.ts]: Why turn this off?

Comment (RedSparr0w) [reply src/scripts/Notifier.ts]: I don't think we can control the names of those parameters for notifications,

```javascript
            allow_dismiss: true,
            newest_on_top: true,
            z_index: 1050,
            url_target: '_blank',
            mouse_over: null,
            icon_type: 'class',
```

It's still turned on for every other file though, just not this one.

Comment (Ishadijcks) [reply src/scripts/Notifier.ts]: Ah fair point, but does it actually trigger there? You'd expect for it to only trip at the definitions

Comment (RedSparr0w) [reply src/scripts/Notifier.ts]: You would think so, but unfortunately it does detect it.

**Pull request merged**